packages/grant-explorer/package.json

```diff
@@ -68,6 +68,7 @@
     "@types/node": "^17.0.25",
     "@types/react": "^18.0.6",
     "@types/react-dom": "^18.0.2",
+    "allo-indexer-client": "github:gitcoinco/allo-indexer-client",
```

Comment:
let's fix it to a specific commit like `github:gitcoinco/allo-indexer-client#commitHash` so that we are sure we don't break the grant explorer if we update the client now that it's still in early development.